ebuild-writing/functions/src_prepare/eapply/text.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<guide self="ebuild-writing/functions/src_prepare/eapply/">
+<chapter>
+<title>Patching with eapply</title>
+<body>
+
+<p>
+The canonical way of applying patches in ebuilds is to use the package
+manager's <c>eapply</c> function.
+</p>
+
+<p>
+<c>eapply</c> was added in EAPI=6. It differs from the previously available
+<c>epatch</c> in several ways:
+</p>
+
+<ul>
+  <li>
+    <c>eapply</c> will not unpack patches for you.
+  </li>
+  <li>
+    The default patch level is <c>-p1</c>. Other patch levels must be specified
+    manually or the command will fail.
+  </li>
+  <li>
+    When specifying a directory, at least one file with a name ending in
+    <c>.patch</c> or <c>.diff</c> must exist or the command fails. Other files
+    are ignored.
+  </li>
+</ul>
+
+<p>
+Note that distributing modified tarballs rather than a vanilla tarball
+and patches is <e>highly</e> discouraged.
```

Comment:
Again, it would reverse it. "Is is highly preferable to distribute patches and apply them over unpacked sources, rather than distribute an already patched tarball."